packages/backend/src/handlers/remote-schemas/resolvers/getBoxProfile/resolver.ts

```diff
@@ -51,12 +51,22 @@ function getImage(
   return '';
 }
 
+type CollectibleAddress = {
+  [network: string]: string;
+};
+
 function getCollectiblesFavourites(
   collectiblesFavorites: Array<CollectiblesFavorite> | null | undefined,
 ) {
   if (!collectiblesFavorites) return [];
-  return collectiblesFavorites.map(({ address, token_id }) => ({
-    address,
-    tokenId: token_id,
-  }));
+  console.log(collectiblesFavorites);
```

Comment:
Do you intend to use this to debug further?

Reply:
Nope! That was a mistake.